### PR TITLE
{phys}[system/system] COMSOL v6.0.0.318

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/d693d759feb41591fbacbd606e6f18bb for a full test report.

**Comment by akesandgren** [comment]
Can benefit from https://github.com/easybuilders/easybuild-easyblocks/pull/2932 to avoid the recursive chmod

**Comment by Micket** [comment]
Apparently, it has a requirement on csh for a particular feature, 
```
--- BEGIN WARNING (1) ---
17:38:42 csh cannot be found.
csh is required by LiveLink for MATLAB in order to call MATLAB functions while solving
--- END WARNING (1) ---
```
sheesh. I'm just adding it to osdeps, don't have time to deal with this

**Comment by Micket** [comment]
Test report by @Micket
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2932
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.7, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/91a55fe1e8f9c8ac001fb53a55c56363 for a full test report.

**Comment by akesandgren** [comment]
Btw, why only 2 parts on the version here? My iso is 6.0.0.354 ...

**Comment by Micket** [comment]
> Btw, why only 2 parts on the version here? My iso is 6.0.0.354 ... (not the same checksum but still)

I'm really not sure if the people at Chalmers renamed the sources or something bad like that.
```
-rw-rw---- 1 c3-builder c3-staff 4.4G Aug 29  2018 /apps/sources/c/COMSOL/COMSOL5.3.1.348_dvd.iso
-rw-rw---- 1 c3-builder c3-staff 4.4G Mar 14  2018 /apps/sources/c/COMSOL/COMSOL53a_dvd.iso
-rw-rw-r-- 1 c3-builder c3-staff 4.0G Dec 19  2019 /apps/sources/c/COMSOL/COMSOL53_dvd.iso
-rw-r--r-- 1 c3-builder c3-staff 5.3G Sep 14  2021 /apps/sources/c/COMSOL/COMSOL56-5.6.0.401.iso
-rw-rw-r-- 1 c3-builder c3-staff 5.2G Sep 14  2021 /apps/sources/c/COMSOL/COMSOL56_dvd-5.6.0.341.iso
-rw-rw-r-- 1 c3-builder c3-staff 5.2G May 11  2021 /apps/sources/c/COMSOL/COMSOL56_dvd.iso
-rw-r--r-- 1 c3-builder c3-staff 6.3G May 12  2022 /apps/sources/c/COMSOL/COMSOL60_dvd.iso
```

are the things I've apparently gotten from them over the years, so, a bit all over the place.

The about section says

COMSOL Multiphysics 6.0 (Build: 318)

so maybe this was supposed to be 6.0.0.318?
I can rename it, what is the exact filenames you have?

**Comment by akesandgren** [comment]
I would have used 6.0.0.138 for that iso.
I can't remember if the original iso file had some other name, I got it from either Lund or Luleå, we don't have it ourselves.
I just have COMSOL-6.0.0.354.iso for the "newest" one we have, but I also have a COMSOL56_dvd.iso in which the "ver" file says 5.6.0.401, that's what I went by when renaming the iso.

File: easybuild/easyconfigs/c/COMSOL/COMSOL-6.0.0.318.eb [hunk 1]
```diff
@@ -10,9 +10,10 @@ problems.
 
 toolchain = SYSTEM
 
-sources = ['COMSOL%(version_major)s%(version_minor)s_dvd.iso']
+# Note: sources from COMSOL are often named poorly. You can view the exact version after installing.
```
Comment (akesandgren) [inline easybuild/easyconfigs/c/COMSOL/COMSOL-6.0.0.318.eb:13]: Perhaps change this to "view the exact version by looking in the xxx file in the iso"

**Comment by Micket** [comment]
Outdated already as 6.1 is out #19063 instead, with @akesandgren comment addressed

**Pull request closed without merging**